config/overlays/fixture-release-0.4/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Same comment, the name of the directory could be e2e-fixture-release-0.4 or we can keep it as it is and then add ironic- in the other directory name

Reply:
My answer to this one would be the same as in the other kustomization, so let's continue the discussion there.